src/pipelines/vaccinations/config.yaml

```
@@ -261,6 +261,18 @@ sources:
       job_group: all
       deferred: true
 
+  # Data sources for CH levels 0 + 1
+  - name: pipelines.vaccinations.ch_authority.SwitzerlandDataSource
+    fetch:
+      - url: "https://www.covid19.admin.ch/api/data/context"
+    test:
+      location_key_match:
+        - '^CH$'
```

Add LI too since we also get that data!
```suggestion
        - '^CH$'
        - '^LI$'
```